api_docs/construct-narrow.md

```diff
@@ -51,7 +51,13 @@ important optimization when fetching messages in certain cases (e.g.
 when [adding the `read` flag to a user's personal
 messages](/api/update-message-flags-for-narrow)).
 
-**Changes**: In Zulip 9.0 (feature level 250), support was added for
+**Changes**: In Zulip 9.0 (feature level 263), narrows gained support for a new
+filter - `with`, taking a message ID as operand. If the message corresponding to
+the message ID exists and is visible to the user, then it replaces any "conversation"
+filters — channel/stream, topic conversation — with filters corresponding to the
+conversation the message is in. Else, the `with` operator is just ignored.
```

Comment:
I think this doesn't clearly specify what happens if no channel/topic operator was present. That's probably OK here

Also, not need to mention `channel/stream`, just say "channel" since that's how we refer to that in prose.

So something like:

> 263), narrows gained support for a new `with` operator, designed to be used in links that should follow a topic across being moved in a single API request.

Reply:
updated